client/js/components/statement/publicStatementLists/DpPublicStatement.vue

```diff
@@ -421,6 +422,10 @@ export default {
         content += `${ln}<br />`
       })
       return DomPurify.sanitize(content)
+    },
+
+    renderUrl (url) {
+      return sanitizeUrl(url)
     }
```

Comment:
I understand the approach but I think this is not neccessary since we are not dealing with user generated input for our links. However I would say we can still keep it but I wonder what we will do with all the other links that are not sanitized in our applications.

Reply:
removed it, thanks for your advice https://github.com/demos-europe/demosplan-core/pull/1243/commits/1b2191b0f5106018db9f98b4e274e82f2ab038f8